skimage/filters/_blur_effect.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import scipy.ndimage as ndi
+
+from ..color import rgb2gray
+from . import sobel
+from ..util import img_as_float
+
+
+__all__ = ['blur_effect']
+
+
+def blur_effect(image, h_size=11, channel_axis=None):
+    """
+    Compute a metric that indicates the strength of blur in an image (0 for no
+    blur, 1 for maximal blur).
+
+    Parameters
+    ----------
+    image : ndarray
+        RGB or grayscale nD image. The input image is converted to grayscale
+        before computing the blur metric.
+    h_size : int, optional
+        Size of the re-blurring filter. Default is 11.
```

Comment:
```suggestion
        Size of the re-blurring filter.
```
We usually don't specify the default value in the docstring.

Reply:
Ok.